nixos/modules/flyingcircus/roles/webproxy.nix

```diff
@@ -56,6 +86,7 @@ in
         Put your configuration into `default.vcl`.
       '';
       "local/varnish/default.vcl.example".text = vcl_example;
+      "varnish.vcl".source = configFile;
```

Comment:
I started with a different pattern where I put store config in /run/current-config/... but maybe this is fine too. However, this could be confusing for users exploring /etc, if they see /etc/varnish.vcl and /etc/local/varnish/default.vcl ...

Reply:
Yes, confusing. Maybe we use /etc/current-config. There is AFAIK no support for run like with enviornment.etc. And I don't want to fiddle around with all those files.